dependencies.yaml

```diff
@@ -212,12 +213,17 @@ dependencies:
           # without CUDA, e.g. 'pip install .'
           - matrix: null
             packages: null
+  test_ucx_py:
+    common:
+      - output_types: conda
+        packages:
+          - ucx-py==0.42.*,>=0.0.0a0
   test_python:
     common:
       - output_types: [conda, requirements, pyproject]
         packages:
           - cloudpickle
-          - dask
+          - rapids-dask-dependency==25.2.*,>=0.0.0a0
```

Comment:
Switch from `dask` to `rapids-dask-dependency` for Dask test dependency. That way it matches however we have Dask constrained elsewhere in RAPIDS